packages/accounts/src/light-account/account.ts

```diff
@@ -32,45 +37,72 @@ import {
 } from "./utils.js";
 
 export type LightAccount<
+  TEntryPointVersion extends EntryPointVersion,
   TSigner extends SmartAccountSigner = SmartAccountSigner
-> = SmartContractAccountWithSigner<"LightAccount", TSigner> & {
+> = SmartContractAccountWithSigner<
+  TEntryPointVersion,
+  "LightAccount",
+  TSigner
+> & {
   getLightAccountVersion: () => Promise<LightAccountVersion>;
   encodeTransferOwnership: (newOwner: Address) => Hex;
   getOwnerAddress: () => Promise<Address>;
 };
 
 export type CreateLightAccountParams<
+  TEntryPointVersion extends EntryPointVersion,
```

Comment:
I don't think we should have added these new generics as the first parameter in these types. These are technically breaking changes